src/libs/actions/Report.ts

```diff
@@ -4362,18 +4362,29 @@ function prepareOnboardingOnyxData(
     return {optimisticData, successData, failureData, guidedSetupData, actorAccountID, selfDMParameters};
 }
 
-function completeOnboarding(
-    engagementChoice: OnboardingPurpose,
-    data: ValueOf<typeof CONST.ONBOARDING_MESSAGES>,
+function completeOnboarding({
+    engagementChoice,
+    data,
```

Comment:
I know this is an existing param, but now that it's a named arg, let's use a better name instead of `data`. I think we can call it `onboardingMessage`.